package/nginx-https.conf

```diff
@@ -10,7 +10,7 @@ server {
     error_log   /var/log/nginx/rmt_https_error.log;
     root        /usr/share/rmt/public;
 
-    ssl_certificate     /usr/share/rmt/ssl/rmt-server.pem;
+    ssl_certificate     /usr/share/rmt/ssl/rmt-server.crt;
```

Comment:
@ikapelyukhin we are suggesting users of nginx to switch to TLS1.2 only. See [line 99](https://build.opensuse.org/package/view_file/server:http/nginx/nginx-1.6.1-default_config.patch?expand=1)

Would that make sense to align RMT config?

Reply:
I'm not sure if TLSv1.2 would work with openssl version that is on SLES12 installation media, but for all modern use cases -- sure, it makes sense.